nav2_controller/src/nav2_controller.cpp

```diff
@@ -147,14 +176,25 @@ ControllerServer::on_shutdown(const rclcpp_lifecycle::State &)
   return nav2_util::CallbackReturn::SUCCESS;
 }
 
-void ControllerServer::followPath()
+void ControllerServer::computeControl()
 {
-  RCLCPP_INFO(get_logger(), "Received a goal, begin following path");
-
+  RCLCPP_INFO(get_logger(), "Received a goal, begin computing control effort.");
+  //TODO STEVE msg string, check and use
```

Comment:
If you're planning on doing this TODO after merging this PR, then could provide more information?

Reply:
Oops, that's a planning artifact, I'll remove it